stock_operating_unit/security/stock_security.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2019 Eficent Business and IT Consulting Services S.L.
+     Copyright 2019 Serpent Consulting Services Pvt. Ltd.
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0) -->
+<odoo>
+    <record id="ir_rule_stock_warehouse_allowed_operating_units" model="ir.rule">
+        <field name="model_id" ref="stock.model_stock_warehouse" />
+        <field name="domain_force">['|',
+            ('operating_unit_id', '=', False),
+            ('operating_unit_id', 'in', user.operating_unit_ids.ids)]
+        </field>
+        <field name="name">Warehouses from allowed operating units</field>
+        <field name="global" eval="True" />
+        <field eval="0" name="perm_unlink" />
+        <field eval="0" name="perm_write" />
+        <field eval="1" name="perm_read" />
+        <field eval="0" name="perm_create" />
+    </record>
+    <record id="ir_rule_stock_picking_type_allowed_operating_units" model="ir.rule">
+        <field name="model_id" ref="stock.model_stock_picking_type" />
+        <field name="domain_force">['|',
+            ('warehouse_id.operating_unit_id','=', False),
+            ('warehouse_id.operating_unit_id','in',user.operating_unit_ids.ids)]
+        </field>
+        <field name="name">Stock Picking Type from allowed operating units</field>
+        <field name="global" eval="True" />
+        <field eval="0" name="perm_unlink" />
+        <field eval="0" name="perm_write" />
+        <field eval="1" name="perm_read" />
+        <field eval="0" name="perm_create" />
+    </record>
+    <record id="ir_rule_stock_location_allowed_operating_units" model="ir.rule">
+        <field name="model_id" ref="stock.model_stock_location" />
+        <field name="domain_force">['|',
+            ('operating_unit_id', 'in',user.operating_unit_ids.ids),
+            ('operating_unit_id','=',False)]
+        </field>
+        <field name="name">Stock locations from allowed operating units</field>
+        <field name="global" eval="True" />
+        <field eval="0" name="perm_unlink" />
+        <field eval="0" name="perm_write" />
+        <field eval="1" name="perm_read" />
+        <field eval="0" name="perm_create" />
+    </record>
+    <record id="ir_rule_stock_move_allowed_operating_units" model="ir.rule">
```

Comment:
could you include a similar rule for stock.move.line? That way we will cover the case for Product Moves. With that I think  this is ok to merge

Reply:
> could you include a similar rule for stock.move.line? That way we will cover the case for Product Moves. With that I think this is ok to merge

I just created a PR to your branch with the changes. https://github.com/BT-nstuder/operating-unit/pull/1 Could you check it? If you agree with the merge, please use the option "Rebase and Merge". That way github will not generate a merge commit. Thank you!

Reply:
Thank you very much. I've merged your PR. Sorry for responding so late.